addons/point_of_sale/static/src/js/Screens/TicketScreen/TicketScreen.js

```diff
@@ -390,6 +392,8 @@ odoo.define('point_of_sale.TicketScreen', function (require) {
                         orderUid: orderline.order.uid,
                         orderBackendId: orderline.order.backendId,
                         orderPartnerId,
+                        // undefined for no tax so that it falls back the the tax in the product
+                        tax_ids: _taxIds.length !== 0 ? _taxIds : undefined,
```

Comment:
why would we fallback on product tax, if it was sold without taxes?

Reply:
You are right.